### PR TITLE
Update manifest when removing PII

### PR DESCRIPTION
**Pull request by billfreeman44**

### CODE REVIEW

**Comment by razor-x** [comment]
@billfreeman44 I had to push some changes for the work I'm doing. Can you take over and update this so that the `channel` objects in the returned `metadata` have a new field `redacted = True/False`? This should be set true for any channel that was modified by the function, and `False` otherwise. Also add to the tests 🙏🏻

**Comment by billfreeman44** [comment]
Should be good. I also added a high-level `redacted` for the manifest, meant to indicate that job id was scrubbed out of there.

**Comment by billfreeman44** [comment]
Does look good? @razor-x

File: pureskillgg_csgo_dsdk/scrubber/scrub_pii.py [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+
+def scrub_csds_pii(manifest: Dict, data: Dict):
+    """Remove events in overtime rounds
+
+    Inputs:
+        manifest (dict): CSDS manifest.
+        data (dict): CSDS data dictionary.
+
+    Returns:
+        Nothing. Data is changed in place.
```
Comment (razor-x) [inline pureskillgg_csgo_dsdk/scrubber/scrub_pii.py]: Update this to document we returned the new manifest.

File: pureskillgg_csgo_dsdk/scrubber/scrub_pii.py [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+def replace_if_exists(
+    data, manifest, channel, column, /, replacement: Union[str, bool, Number] = REDACTED
+):
+    """Replace a column with a certain value if it exists"""
+    if data_exists(data, channel, column):
+        data[channel][column] = replacement
+        channel_index, column_index = get_manifest_indexes(manifest, channel, column)
+        manifest["channels"][channel_index]["columns"][column_index][
+            "origin"
+        ] += "-redacted"
```
Comment (razor-x) [inline pureskillgg_csgo_dsdk/scrubber/scrub_pii.py:73]: Should this use `REDACTED`? Anywhere else?

```suggestion
        ] += f"-{REDACTED}"
```

**Review by razor-x — CHANGES_REQUESTED** [review]
Found a few random things, otherwise good to merge after those.

File: pureskillgg_csgo_dsdk/scrubber/scrub_pii.py (hunk 1 above)
Comment (billfreeman44) [reply pureskillgg_csgo_dsdk/scrubber/scrub_pii.py]: Done

File: pureskillgg_csgo_dsdk/scrubber/scrub_pii.py (hunk 2 above)
Comment (billfreeman44) [reply pureskillgg_csgo_dsdk/scrubber/scrub_pii.py:73]: I don't think it should. I think this is more like a key where `REDACTED` is the value assigned to the overwritten data.

Comment (billfreeman44) [reply pureskillgg_csgo_dsdk/scrubber/scrub_pii.py:73]: Though I'm not super set on this, if you want I can change it to use f string

**Review by razor-x — APPROVED** [review]

**Pull request merged**